ChangeLog.d/7764.txt

```diff
@@ -1,4 +1,4 @@
 Features
-   * mbedtls_ecc_group_to_psa() and mbedtls_ecc_group_from_psa() helper
-     functions were added to convert from mbedtls_ecp_group_id to
-     psa_ecc_family_t and curve's bit size and viceversa, respectively.
+   * Add functions mbedtls_ecc_group_to_psa() and mbedtls_ecc_group_from_psa()
+     to convert between mbedtls_ecp_group_id and psa_ecc_family_t and curve's
+     bit size.
```

Comment:
“Between X and Y and Z” looks weird: the natural reading is that there are conversions between any two of X, Y and Z. I think you don't need to mention the bit size here at all: it's just between `mbedtls_ecp_group_id` and `psa_ecc_family_t`, or between mbedtls and PSA curve identifiers.